app/src/main/java/org/p2p/wallet/auth/interactor/SignUpFlowDataCache.kt

```diff
@@ -0,0 +1,58 @@
+package org.p2p.wallet.auth.interactor
+
+import com.google.gson.JsonObject
+import org.p2p.solanaj.core.Account
+import org.p2p.wallet.auth.model.Web3AuthSignUpResponse
+import org.p2p.wallet.auth.web3authsdk.UserSignUpDetailsStorage
+import org.p2p.wallet.utils.Base58String
+import org.p2p.wallet.utils.emptyString
+import org.p2p.wallet.utils.toBase58Instance
+import timber.log.Timber
+
+private val TAG = SignUpFlowDataCache::class.simpleName.orEmpty()
+
+class SignUpFlowDataCache(
```

Comment:
couldn't create something more useful than this thing
it aggregates all the data created due to CreateWallet flow that we don't want to keep in shared prefs

it's okay to lose all this when user exists the app, it will be filled when he starts anew

Reply:
Why you don't use for this LocalRepository ?